cnf-tests/testsuites/e2esuite/sctp/sctp_sriov.go

```diff
@@ -39,6 +39,9 @@ var _ = Describe("[sriov] SCTP integration", func() {
 		err := namespaces.Create(TestNamespace, client.Client)
 		Expect(err).ToNot(HaveOccurred())
 
+		err = namespaces.Create(sriovnamespaces.Test, client.Client)
```

Comment:
Why you need this here?

Reply:
we need this namespace to exist before we run `sriovcluster.DiscoverSriov` as it start a pod on that namespace to read the secure-boot status from the node

Reply:
mind adding a comment for that?